pywinauto/unittests/test_keyboard.py

```diff
@@ -283,7 +283,7 @@ def testShiftModifier(self):
         received = self.receive_text()
         self.assertEqual("A", received)
 
-    if sys.platform != 'win32':
+    if sys.platform.startswith('linux'):
```

Comment:
Not sure we need all such conditions to change. It was already changed somewhere in macOS branch (see PR #841). So some tests are running on both Linux and macOS with the same code (at least for mouse, keyboard and clipboard). I'd suggest to leave the fix only for the code I've commented on.

Reply:
Maybe change the most recent commit with force push to avoid conflicts in neighbor PR.